deploy/deploy-sota-local.yml

```diff
@@ -78,9 +78,7 @@ services:
     ports:
       - 80:9000
     env:
-      CORE_PORT: "8080"
-      CORE_HOST: "core"
-      RESOLVER_PORT: "8081"
-      RESOLVER_HOST: "resolver"
+      CORE_API_URI: "http://core:8080"
+      RESOLVER_API_URI: "http://resolver:8081"
```

Comment:
It does look cleaner true, it's just we got burned with this before with things swithing between expecting URIs and expecting hosts and ports, and then expecting to find a protocol or not. It's probably ok as URIs, but for reasons I'm struggling to articulate I'd be more comfortable with hosts and ports. Since I can't say for definite I'll leave it up to you.